spec/controllers/coordinator_controller_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'rails_helper'
+
+RSpec.describe CoordinatorsController, type: :controller do
+  describe 'endpoints' do
+    before do
+      sign_in_auth
+    end
+
+    describe 'GET #show' do
+      before do
+        @coordinator = create(:coordinator)
+      end
+
+      it 'populates the specified coordinator' do
+        get :show, id: @coordinator
+        expect(assigns(:coordinator)).to eq(@coordinator)
+      end
+
+      it 'populates an enrollment' do
```

Comment:
It doesn't populate an enrollment! But it *does* populate students and tutors!
I suggest moving the `students` and `tutors` methods into the `private` part of this controller and then testing that the `show` view populates students and tutors. The easiest way to do this is probably to create a factory for it, but you can also definitely set it up manually! Let me know if I can help.

To make private methods, you just move the method definitions underneath the `private` keyword near the bottom of the controller, so the methods would move either directly under or directly over the `coordinator_params` method.